tests/testthat/test-knitr-options.R

```diff
@@ -28,6 +28,25 @@ test_that("`tidyverse_quiet` works", {
   expect_match(ret, "Attaching", all = FALSE)
 })
 
+test_that("`tidyverse_quiet` works for tidymodels", {
+  skip_on_cran()
+  skip_if_not_installed("tidymodels")
+
+  ret <- reprex(
+    input = sprintf("library(%s)\n", "tidymodels"),
```

Comment:
Is there a reason to use `sprintf()` as opposed the literal `"library(tidymodels)\n"`? Ditto for next expectation.

Reply:
I will be honest that I don't see any reason for `sprintf()`, but I matched how the tests for tidyverse were done in the same file. I am happy to switch just tidymodels, or both. What is your preference?

Reply:
Ha! I have no idea why I did that. Can you do the simpler thing everywhere? Seems like it should work and would be an improvement.